editor.planx.uk/src/@planx/components/PlanningConstraints/Public.tsx

```diff
@@ -30,8 +35,10 @@ function Component(props: Props) {
     (state) => state.computePassport().data?.["property.boundary.site"]
   );
   const { x, y, longitude, latitude, usrn } =
-    useStore((state) => state.computePassport().data?._address) || {};
-  const showGraphError: boolean = !x && !y && !longitude && !latitude;
+    (useStore(
+      (state) => state.computePassport().data?._address
+    ) as SiteAddress) || {};
+  const showGraphError = !x && !y && !longitude && !latitude;
```

Comment:
We show the graph error if all of these are missing - should it be if any of them are missing?

Reply:
Fair question - I think you're right that `||` makes more sense than `&&` here even though these fields are all required on our minimum address type. Either case should still only happen when: 1) planning constraints isn't positioned after find property, or 2) there's been an error in OS source data :ok_hand: